### ISSUE TITLE
plugin causes initialize not to be called

### ISSUE THREAD

**Issue by jamesdixon** [bsiddiqui/bookshelf-bcrypt#2]
Hi,

First off, great plugin!

I attempted to use it and while it worked, it prevented the `initialize()` method on my models from calling. You can easily reproduce it by changing `test/db/models/user.js` to the following:

``` js
module.exports = db.bookshelf.model('User', {
  tableName: 'users',
  bcrypt: { field: 'password' },
  initialize: function () {
    console.log('test')
  }
})
```

You'll notice `test` isn't logged to the console when running the tests. However, if you comment out `bcrypt: { field: 'password' },`, you'll see that `test` is then written to the console.

**Comment by alanhoff** [comment]
Thx for the reporting @jamesdixon, it should be fixed in `2.0.1` :-)

**Comment by jamesdixon** [comment]
No problem. Thx, @alanhoff !
